VideoLocker/res/layout/fragment_add_comment.xml

```diff
@@ -58,7 +58,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
-                android:gravity="top"
+                android:gravity="start"
```

Comment:
I thought so but this fixed the rendering in the Android Studio preview. Maybe it's not actually needed though, I can check.